File: android/app/src/main/java/com/stockswing/app/model/RealtimeQuote.kt
package com.stockswing.app.model

/** TWSE MIS API 的即時行情快照 */
data class RealtimeQuote(
    val code: String,
    val name: String,
    val price: Double,
    val open: Double,
    val high: Double,
    val low: Double,
    val prevClose: Double,
    val chgPct: Double,
    val totalVolLots: Long,
    val updateTime: String,
)

File: ios/StockSwing/Models/RealtimeQuote.swift
import Foundation

struct RealtimeQuote: Codable {
    let code: String
    let name: String
    let price: Double
    let open: Double
    let high: Double
    let low: Double
    let prevClose: Double
    let chgPct: Double
    let totalVolLots: Int64
    let updateTime: String
}
